src/index.html

```diff
@@ -204,37 +204,73 @@ <h2 class="title__heading">
+                  <h3 class="project__heading">Hand Watch</h3>
+                  <p class="project__description">ui/ux design</p>
+                </div>
+              </a>
             </li>
-            <li class="projects__project project">
-              <!-- Figure it out by your-self -->
+            <li class="projects__project project" data-cat="uiux">
+              <a href="#" class="project__link">
+                <img src="images/projects/project-2.jpg" alt="Project Name" class="project__image">
+                <div class="project__hover">
+                  <h3 class="project__heading">Hand Watch</h3>
+                  <p class="project__description">ui/ux design</p>
+                </div>
+              </a>
             </li>
-            <li class="projects__project project">
-              <!-- Figure it out by your-self -->
+            <li class="projects__project project" data-cat="design">
+              <a href="#" class="project__link">
+                <img src="images/projects/project-3.jpg" alt="Project Name" class="project__image">
+                <div class="project__hover">
+                  <h3 class="project__heading">Hand Watch</h3>
+                  <p class="project__description">ui/ux design</p>
+                </div>
+              </a>
             </li>
-            <li class="projects__project project">
-              <!-- Figure it out by your-self -->
+            <li class="projects__project project" data-cat="mockups">
+              <a href="#" class="project__link">
+                <img src="images/projects/project-4.jpg" alt="Project Name" class="project__image">
+                <div class="project__hover">
+                  <h3 class="project__heading">Hand Watch</h3>
+                  <p class="project__description">ui/ux design</p>
+                </div>
+              </a>
             </li>
-            <li class="projects__project project">
-              <!-- Figure it out by your-self -->
+            <li class="projects__project project" data-cat="uiux">
+              <a href="#" class="project__link">
+                <img src="images/projects/project-5.jpg" alt="Project Name" class="project__image">
+                <div class="project__hover">
+                  <h3 class="project__heading">Hand Watch</h3>
+                  <p class="project__description">ui/ux design</p>
+                </div>
+              </a>
             </li>
-            <li class="projects__project project">
-              <!-- Figure it out by your-self -->
+            <li class="projects__project project" data-cat="mockups">
+              <a href="#" class="project__link">
+                <img src="images/projects/project-6.jpg" alt="Project Name" class="project__image">
+                <div class="project__hover">
+                  <h3 class="project__heading">Hand Watch</h3>
+                  <p class="project__description">ui/ux design</p>
+                </div>
```

Comment:
It's better to describe what's the block about instead what it does. I understand that it has hover effect, but in reality it's something like `project__title-wrap`... See, with your class name I cannot even understand what the block is.

Reply:
The same problem with the `project__link`.